spec/models/hearing_spec.rb

```diff
@@ -4,6 +4,10 @@
   let(:hearing) { described_class.new(body: '{}') }
   subject { hearing }
 
+  context 'the hearing has not been resulted' do
+    it { expect(subject).to_not be_persisted }
+  end
+
```

Comment:
Not entirely sure if this adds value to the `resulted?` check itself, so I would suggest we take this off.